packages/video_player/video_player_platform_interface/lib/method_channel_video_player.dart

```diff
@@ -78,6 +78,32 @@ class MethodChannelVideoPlayer extends VideoPlayerPlatform {
       ..volume = volume);
   }
 
+  @override
+  Future<List<Object?>> getAvailableAudioTracksList(int textureId) async {
+    final AudioTrackMessage audioTrackMessage = await _api.getAvailableAudioTracksList(
+      TextureMessage()..textureId = textureId,
+    );
+    return audioTrackMessage.audioTrackNames!;
```

Comment:
не боишься нал эксептиона, может нужен какой-нибудь чек?

Reply:
зачем вообще бэнг оператор, если это проперти вроде как не налубол?

Reply:
getAvailableAudioTracksList возвращает Future<List<Object?>>, поэтому конечно проперти налабл

и так как это из натива прилетает - невозможно не налабл тип сделать (либо dynamic, либо Object?) 

но я не боюсь, тк из натива возвращается лист non-nullable String

Reply:
`Error: A value of type 'List<String?>?' can't be returned from an async function with return type 'Future<List<String?>>' because 'List<String?>?' is nullable and 'Future<List<String?>>' isn't.`

ну и бэнг конечно нужен, тк `_api.getAvailableAudioTracksList` возвращает `Future<List<Object?>>`